src/map_server/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```

Comment:
Please follow the formatting conventions for the CMakeLists.txt files (see other examples in the source tree).